native/service/governance/side_chain_manager/side_chain_manager.go

```diff
@@ -431,3 +447,122 @@ func SetBtcTxParam(native *native.NativeService) ([]byte, error) {
 	}
 	return utils.BYTE_TRUE, nil
 }
+
+func RegisterAssetMap(native *native.NativeService) ([]byte, error) {
+	params := new(RegisterAssetParam)
+	if err := params.Deserialization(common.NewZeroCopySource(native.GetInput())); err != nil {
+		return utils.BYTE_FALSE, fmt.Errorf("RegisterAssetMap, contract params deserialize error: %v", err)
+	}
+
+	//check witness
+	err := utils.ValidateOwner(native, params.OperatorAddress)
+	if err != nil {
+		return utils.BYTE_FALSE, fmt.Errorf("RegisterAssetMap, checkWitness error: %v", err)
+	}
+
+	operatorAddress, err := GetOperatorAddress(native)
+	if err != nil {
+		return utils.BYTE_FALSE, fmt.Errorf("RegisterAssetMap, GetOperatorAddress error: %v", err)
+	}
+	if operatorAddress == common.ADDRESS_EMPTY {
+		PutOperatorAddress(native, params.OperatorAddress)
```

Comment:
Operator may better to be set when side chain is registered, and isolated with chainIDs

Reply:
That's a good idea